File: Java/program9.java

/*
    Step 1 : Understand the problem statement
    Step 2 : Write the algorithm
    Step 3 : Decide the programming language
    Step 4 : Write the program
    Step 5 : Test the program
*/


/*
    Algorithm

    START
        Accept first number as no1
        Accept second number as no2
        If the input is negative then convert it into positive
        Perform Addition of no1 & no2
        Display the addition on screen
    STOP
*/
import java.util.Scanner;
class Add
{   
    public float sum (float  a , float  b) 
    {
        float sum = 0;

        // updator 
        if(a < 0.0f)
    {
        a = -a;
    }

    if(b < 0.0f)
    {
        b = -b;
    }

    sum = a + b;    // Business Logic
    
    return sum;
    }
            
}
    

public class program9 {
    public static void main (String A[])
    {
        Scanner sobj = new Scanner(System.in);

        float i = 0.0f;
        float j = 0.0f;


         System.out.println("Enter the first nos ");
         i = sobj.nextFloat();

         System.out.println("Enter the sec nos ");
         j = sobj.nextFloat();

         float ans = 0;

         Add aobj = new Add();
         ans = aobj.sum(i,j);
         System.out.println("Anser is " + ans);
    }
}

File: Go/program9.go
package main

import "fmt"

func AdditionTwoNumbers(nos1 float64, nos2 float64) float64 {
	var sum float64 = nos1 + nos2

	return sum

}
func main() {

	var nos1 float64
	var nos2 float64
	fmt.Println("Enter the nos")
	fmt.Scan(&nos1)

	fmt.Println("Enter the nos")
	fmt.Scan(&nos2)
	var sum float64 = nos1 + nos2

	sum = AdditionTwoNumbers(nos1, nos2)
	fmt.Println(sum)

	if nos1 < 0 {
		nos1 = -nos1
	}
	if nos2 < 0 {
		nos2 = -nos2
	}

}
